static/src/javascripts/projects/commercial/modules/article-body-adverts.ts

```diff
@@ -147,11 +152,22 @@ const addDesktopInlineAds = (isInline1: boolean): Promise<boolean> => {
 		filter: filterNearbyCandidates(adSizes.mpu.height),
 	};
 
+	let minAbove = 1000;
+
+	if (isPaidContent) {
+		minAbove += 600;
+	}
+
+	/* On old articles without a main image or articles with a showcase main image, inline2 can sometimes overlap the most viewed articles */
+	if (!hasImages || hasShowcaseMainElement) {
+		minAbove += 100;
+	}
+
 	// For any other inline
 	const relaxedRules: SpacefinderRules = {
 		bodySelector: articleBodySelector,
 		slotSelector: ' > p',
-		minAbove: isPaidContent ? 1600 : 1000,
+		minAbove,
```

Comment:
Is it possible to add to the `selectors` in order to conditionally change `minAbove` depending on the elements in the dom?

Or are they for something else?

Reply:
I think they're for setting `min(Above|Below)` from elements in the article body?